src/Patches/TcpNetServer.cs

```diff
@@ -1,3 +1,4 @@
+using System.Collections.Concurrent;
```

Comment:
I would prefer to have this line after "using RPVoiceChat.Utils;" to be cleaner but it's a small detail, i will not block this PR for that (i will clean later that with cleaning automation)